src/styles/_catalog.scss

```diff
@@ -14,9 +18,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+
+    &:hover {
+      color: #34568b;
+      cursor: pointer;
```

Comment:
Don't set cursor prop in the :hover. It's the same case as with setting transition in the :hover